src/Illuminate/Contracts/Cache/Repository.php

```diff
@@ -38,7 +38,7 @@ public function pull($key, $default = null);
      *
      * @param  string  $key
      * @param  mixed   $value
-     * @param  \DateTimeInterface|\DateInterval|float|int  $minutes
+     * @param  \DateTimeInterface|\DateInterval|float|int|null  $minutes
```

Comment:
This is a breaking change for downstream implementors

Reply:
You're right. I'll send this in to master instead.